test/helpers.bash

```diff
@@ -124,6 +138,7 @@ CRIO_CONFIG="$TESTDIR/crio.conf"
 CRIO_CNI_CONFIG="$TESTDIR/cni/net.d/"
 CRIO_LOG="$TESTDIR/crio.log"
 
+# CONTAINER_CNI_CONFIG_DIR=${CONTAINER_CNI_CONFIG_DIR:-"$CRIO_CNI_CONFIG"}
```

Comment:
Was this left over from earlier changes?

Reply:
@umohnani8  Prior to commenting this out, it was failing 3 tests. There were two tests in network.bats that were failing, so even though cni_config_dir isn't a slice, I tried omitting this and it passed the tests, so only one fail now. Planned to delete it in the next commit when hopefully all tests pass.